docs/data-sources/client.md

```diff
@@ -32,7 +32,6 @@ data "auth0_client" "some-client-by-id" {
 
 ### Read-Only
 
-- `addons` (List of Object) Addons enabled for this client and their associated configurations. (see [below for nested schema](#nestedatt--addons))
```

Comment:
Addons are temporarily removed within the v1 branch as to not include too many changes within this PR. The next follow up PR will be adjusting the client addons schema with the proper terraform schema and go types.